ownerdelete.user.js

```diff
@@ -310,10 +323,10 @@ dry.once("load", () => {
       mi.addEventListener("click", function() {
         const known = new Set();
         dry.exts.filelistManager.filelist.filelist.forEach(e => {
-          const k = `${e.size}/${e.name}`;
+          const k = e.checksum;
```

Comment:
The checksum might not be available yet, or at all, at the getinfo call might not have returned or errored out.
At the very least, you gotta check for that.
Better yet, you'd store in e.checksum the actual getinfo promise, so you can wait for it (and display some notice that the list is still being populated if it takes too long)

Reply:
PS: if checksum is null or undefined, you' potentially select a bunch of unrelated files (all having null/undefined) which aren't actual dupes, so this is a major bug.

Reply:
I added early if case that check if promise pool responsible for assigning checksums to files is still processing. If not, than we can select dupes.

Reply:
STILL should make sure the API actually returned something sensible